_docs/setup/mail/routing.md

```diff
@@ -34,49 +34,223 @@ You can configure rules to automatically route new mail to groups based on messa
+        or sender_num_spam >= 10
+      }}
+  then:
+    group: Special Handling
+    owner: KinaHalpue
+{% endraw %}
+{% endhighlight %}
+
+### recipients
+
+```recipients``` routes based on the address(es) the mail is sent to. You can use full addresses. prefixes or domains. You can include multiple addresses in a comma-separated list or in a line-separated list with the ```@list``` annotation.
+
+{% highlight cerb %}
+  rule/billing:
+    if:
+      recipients: billing@, receipts@
+    then:
+      group: Billing
+  rule/brand2:
+    if:
+      recipients@list: 
+        help@brand2.example
+        sales@brand2.example
+    then:
+      group: Brand 2
+{% endhighlight %}
+
+### sender_email
+```sender_email``` routes based on the address(es) of the sender.  You can use full addresses. prefixes or domains. You can include multiple addresses in a comma-separated list or in a line-separated list with the ```@list``` annotation.
+
+{% highlight cerb %}
+  rule/vip:
+    if:
+      sender_email: *@vipcustomer.example
+    then:
+      group: VIP
+      importance: 90
+  {% endhighlight %}
+
+### spam_score
+```spam_score``` routes based on the assigned spam score of an email. You can use ```<```,```>``` and ```=``` operators to specify a "greater than" or "less than" value.
+
+{% highlight cerb %}
+  rule/spam:
+    if:
+      spam_score: >=80%
+    then:
+      group: Spam
+{% endhighlight %}
+
+### subject
+```subject``` routes based on the text of the email subject line.
+
+{% highlight cerb %}
+rule/bugs:
+  if:
+    subject: this is a bug
+  then:
+    group: Development
+{% endhighlight %}
+
+### body
+```body``` routes based on content of the email message body.
+
+{% highlight cerb %}
+rule/campaign:
+  if:
+    body: *I'm responding to your marketing campaign*
+  then:
+    group: Marketing
+{% endhighlight %}
+
+### header
+```header``` routes based on content from the message header
+
+{% highlight cerb %}
+rule/autoreplies:
+  if:
+    header: 
+      Auto-Submitted:"autoreplied"
+  then:
+    group: Spam
+    bucket: Autoreplies
+{% endhighlight %}
+
+## Actions
+
+Actions are what you want to occur if the conditions are met. You can have multiple actions for each routing rule.
+
+### bucket
+```bucket``` sets a bucket to route the ticket.
+
+{% highlight cerb %}
+rule/bugs:
+  if/email:
+    recipients: bugs@
+  if/subj:
+    subject: bug report
   then:
     group: Development
+    bucket: Bugs 
+{% endhighlight %}
+
+### commnent
+  ```comment``` adds a comment with the given text to the resulting ticket.
 
-rule/sales:
+{% highlight cerb %}
+rule/vip:
   if:
-    recipients: sales@
+    sender_email: *@vipcustomer.example
   then:
-    group: Sales
+    group: VIP
+    importance: 90
+    comment: This is a high value customer. Please respond rapidly.
+{% endhighlight %}
+
+### group
+```group``` defines the group a ticket will be moved to. If you do not also use ```bucket`` the ticket will be placed in the group's Inbox.`
```

Comment:
Trailing (`) on this line. Be sure you're previewing the final output in the local Jekyll site.